mne/utils/tests/test_config.py

```diff
@@ -87,3 +87,20 @@ def test_sys_info():
 
     if platform.system() == 'Darwin':
         assert 'Platform:      macOS-' in out
+
+
+def test_get_subjects_dir(tmpdir):
+    """Test get_subjects_dir()."""
+    # String
+    subjects_dir = '/foo'
+    assert get_subjects_dir(subjects_dir) == subjects_dir
+
+    # Path
+    subjects_dir = Path('/foo')
+    assert get_subjects_dir(subjects_dir) == str(subjects_dir)
+
+    # `None` in config
+    subjects_dir = None
+    set_config(key='SUBJECTS_DIR', value=subjects_dir, home_dir=tmpdir,
+               set_env=True)
```

Comment:
Tests should leave global state unmodified. Using set_env=True here will modify is.environ, hence possible affecting subsequent tests (for example if someone calls SUBJECTS_DIR=bad pytest...). You should use monkeypatch.delenv with raising=False instead.

Reply:
Thanks @larsoner, I didn't know about `monkeypatch.*`, good to know!